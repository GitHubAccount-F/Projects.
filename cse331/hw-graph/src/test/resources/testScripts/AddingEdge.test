#Tests to see if edges were added properly between two nodes

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2

#Adds nodes to graphs
AddNode graph1 one
AddNode graph1 two
AddNode graph2 one
AddNode graph2 two

#Lists the nodes
ListNodes graph1
ListNodes graph2

#Adds the edge(s)
AddEdge graph2 one two e1

#Check to see if edges were added
ListChildren graph1 one
ListChildren graph2 one