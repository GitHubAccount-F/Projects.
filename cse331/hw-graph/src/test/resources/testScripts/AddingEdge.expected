#Tests to see if edges were added properly between two nodes

#Creates empty graphs
created graph graph1
created graph graph2

#Adds nodes to graphs
added node one to graph1
added node two to graph1
added node one to graph2
added node two to graph2

#Lists the nodes
graph1 contains: one two
graph2 contains: one two

#Adds the edge(s)
added edge e1 from one to two in graph2

#Check to see if edges were added
the children of one in graph1 are:
the children of one in graph2 are: two(e1)